packages/react/src/components/UIShell/SideNavDivider.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright IBM Corp. 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { settings } from 'carbon-components';
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const { prefix } = settings;
+
+const SideNavDivider = ({ className }) => {
```

Comment:
Small style-nit, we try to use the function declaration form based on the [styled guide](https://github.com/carbon-design-system/carbon/blob/master/docs/style.md#writing-a-component) 👀

Reply:
👍  Updated